src/discussions/posts/post/PostHeader.jsx

```diff
@@ -110,12 +117,30 @@ function PostHeader({
               </div>
             )
             : <h4 className="mb-0" style={{ lineHeight: '28px' }} aria-level="1" tabIndex="-1" accessKey="h">{post.title}</h4>}
-          <AuthorLabel
-            author={post.author || intl.formatMessage(messages.anonymous)}
-            authorLabel={post.authorLabel}
-            labelColor={authorLabelColor && `text-${authorLabelColor}`}
-            linkToProfile
-          />
+          <div className="align-items-center d-flex flex-row">
+            <AuthorLabel
+              author={post.author || intl.formatMessage(messages.anonymous)}
+              authorLabel={post.authorLabel}
+              labelColor={authorLabelColor && `text-${authorLabelColor}`}
+              linkToProfile
+            />
+            {topicContext && topic && (
+            <div className="mr-3 font-size-14 font-style-normal font-family-inter font-weight-400 d-flex">
+              <span className="text-gray-400">{`${intl.formatMessage(messages.relatedTo)} `}</span>
+              <Hyperlink
+                destination={topicContext.unitLink}
+                target="_top"
+              >
+                <div className="mw-md ml-1 d-flex flex-row">
+                  <span className="container-mw-xs text-truncate"> {`${topicContext.ChapterName} `}</span>
+                  <span className="container-mw-xs text-truncate"> {` / ${topicContext.verticalName}`} {' '}</span>
+                  <span className="container-mw-xs text-truncate"> {` / ${topicContext.unitName}`}</span>
+                </div>
+
+              </Hyperlink>
```

Comment:
this PR is only fixing the design for when in-context is enabled? what about use case when in-context discussion are bot enabled?